src/main/java/org/mitre/synthea/world/concepts/HealthRecord.java

```diff
@@ -176,6 +177,41 @@ public CarePlan(long time, String type) {
     }
   }
 
+  public class ImagingStudy extends Entry {
+    public List<Series> series;
+
+    public ImagingStudy(long time, String type) {
+      super(time, type);
+      this.series = new ArrayList<Series>();
+    }
+
+    /**
+     * ImagingStudy.Series represents a series of images that were taken of
+     * a specific part of the body.
+     */
+    public class Series {
```

Comment:
My gut feeling is that this fits better as a "Component" rather than in the health record, but it's not critical to change it.

Reply:
I took "Components" to be more generic JSON elements that are shared by multiple states. That seems true of all of the existing ones anyway. Since `Series` and `Instance` are really specific to just an `ImagingStudy` and no other states, I think it's best to leave them where they are.